src/styled/Components/Header.ts

```diff
@@ -534,7 +534,10 @@ export const NetworkItem = styled(motion.li)`
     }
 `;
 
-export const ChainNameStatus = styled.div`
+export const ChainNameStatus = styled.div<{ active: boolean }>`
+    display: flex;
+    flex-direction: row;
```

Comment:
row is the default flex-direction, so not needed here. 


We could possible refactor this as a FlexContainer and just use all the props, but not crucial. would be something like 

```
<FlexContainer
alignItems='center'
padding='0.6rem 0'
...
/>

```

Reply:
There are a few stylings being applied here that I would rather keep out of the jsx statement
`
    &:hover,
    &:focus-visible {
        filter: drop-shadow(0 0 5px var(--text1));
    }

    img {
        margin-right: 0.5em;
        vertical-align: middle;
    }

    border-bottom: ${(props) =>
        props.active ? '1px solid var(--accent1)' : ''};`